### PR TITLE
RFC-49: Community Merge

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

### CODE REVIEW

**Comment by Xuanwo** [comment]
ping all our community members for reviewing.

@beyondstorage/beyond-tp-committer 
@beyondstorage/beyond-tp-maintainer 
@beyondstorage/beyond-tp-reviewer 
@beyondstorage/go-storage-committer 
@beyondstorage/go-storage-maintainer 
@beyondstorage/go-storage-reviewer

File: docs/rfcs/49-community-merge.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+---
+
+After all those merging, we will have:
+
+- campfire
+- go-storage
+- beyond-tp
+- gsp-xxx
+- zh-hans(zh-hant, ru and so on)
```
Comment (JinnyYi) [inline docs/rfcs/49-community-merge.md:40]: Will rooms for requirement discussion or matchmaking like https://matrix.to/#/#beyondstorage@project-qingcloud-lowcode:matrix.org be reserved?

Comment (Xuanwo) [reply docs/rfcs/49-community-merge.md:40]: Yes.

**Review by JinnyYi — APPROVED** [review]

**Review by xxchan — APPROVED** [review]

File: docs/rfcs/49-community-merge.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+- Author: Xuanwo <github@xuanwo.io>
+- Start Date: 2021-09-23
+- RFC PR: [beyondstorage/community#49](https://github.com/beyondstorage/community/issues/49)
+- Tracking Issue: [beyondstorage/community#0](https://github.com/beyondstorage/community/issues/0)
```
Comment (Xuanwo) [inline docs/rfcs/49-community-merge.md:4]:
```suggestion
- Tracking Issue: [beyondstorage/community#50](https://github.com/beyondstorage/community/issues/50)
```

**Pull request merged**